Makefile

```diff
@@ -126,6 +126,7 @@ endif
 # clang-format all source files
 format:
 	$(QUIET) find src include -type f \( -name '*.c' -o -name '*.h' \) -exec $(FORMAT) {} +
+	$(QUIET) tools/newlines.sh
```

Comment:
Does this work as expected? I don't actually know how make works at all.

Reply:
You need to sh tools/newlines.sh, or add a shebang to it

Reply:
Okay, I think it needs to be `#!/bin/bash` so I'll add that.

Reply:
Sounds good, but please use `#!/usr/bin/env bash` for better distro compatibility